File: Source/Aladdin.CAPI.ANSI.Java/src/aladdin/asn1/ansi/x962/SpecifiedMultiples.java
package aladdin.asn1.ansi.x962;
import aladdin.asn1.*; 
import java.io.*;

////////////////////////////////////////////////////////////////////////////////
// SpecifiedMultiples ::= SEQUENCE OF SpecifiedMultiple
////////////////////////////////////////////////////////////////////////////////
public final class SpecifiedMultiples extends Sequence<SpecifiedMultiple>
{
    private static final long serialVersionUID = -5711293012914758220L;
    
	// конструктор при раскодировании
	public SpecifiedMultiples(IEncodable encodable) throws IOException 
	{
		super(SpecifiedMultiple.class, encodable); 
	} 
	// конструктор при закодировании
	public SpecifiedMultiples(SpecifiedMultiple... values) 
	{
		super(SpecifiedMultiple.class, values); 
	} 
}

File: Source/Aladdin.CAPI.ANSI/ASN1/ANSI/X962/SpecifiedMultiples.cs
using System; 
using System.Runtime.Serialization;

namespace Aladdin.ASN1.ANSI.X962
{
    ////////////////////////////////////////////////////////////////////////////////
    // SpecifiedMultiples ::= SEQUENCE OF SpecifiedMultiple
    ////////////////////////////////////////////////////////////////////////////////
	[Serializable]
    public class SpecifiedMultiples : Sequence<SpecifiedMultiple>
    {
		// конструктор при сериализации
        protected SpecifiedMultiples(SerializationInfo info, StreamingContext context) : base(info, context) {}

	    // конструктор при раскодировании
	    public SpecifiedMultiples(IEncodable encodable) : base(encodable) {}  

	    // конструктор при закодировании
	    public SpecifiedMultiples(params SpecifiedMultiple[] values) : base(values) {} 
    }
}